schema.ts

```diff
@@ -2081,10 +2160,77 @@ function normalizeIndexProperty(value: IndexedProperty, normalizedProperties: Ob
   return normalizeSchemaPropertyIdentifier(value, normalizedProperties);
 }
 
+function normalizeContentCategorization(
+  value: ContentCategorization,
+  normalizedProperties: ObjectSchemaProperties,
+): ContentCategorization {
+  switch (value.type) {
+    case ContentCategorizationType.Messaging:
+    case ContentCategorizationType.Document:
+    case ContentCategorizationType.Comment:
+      const {type, ...rest} = value;
+      ensureNever<keyof typeof rest>();
+      return {type};
+    case ContentCategorizationType.Email: {
+      const {
+        type: emailType,
+        toProperty,
+        fromProperty,
+        subjectProperty,
+        htmlBodyProperty,
+        plainTextBodyProperty,
+        ...emailRest
```

Comment:
Just FYI if you wanted to reuse the `rest` name here, I think it would work if you put curly brackets up above in the case on line 2170, the brackets create a nested namespace frame.